packages/lesswrong/lib/react-headroom/index.tsx

```diff
@@ -166,18 +191,18 @@ export default class Headroom extends Component {
 
     return (parent === window || parent === document.body)
       ? this.getViewportHeight()
-      : this.getElementPhysicalHeight(parent)
+      : this.getElementPhysicalHeight(parent as HTMLElement)
```

Comment:
This, however, was easier to forcibly type cast than leverage language-native type-narrowing functionality.